ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -146,6 +146,17 @@ tests:
     test:
     - ref: windows-e2e-operator-test-byoh
     workflow: windows-e2e-operator-test-platform-none-vsphere
+- as: wicd-unit-vsphere
+  steps:
+    cluster_profile: vsphere-platform-none
+    post:
+    - chain: upi-vsphere-windows-post
+    pre:
+    - ref: ipi-install-rbac
+    - chain: upi-conf-vsphere-platform-none-ovn
```

Comment:
This chain includes unnecessary steps, for example, `ovn-conf-hybrid-manifest-with-custom-vxlan-port`.

Reply:
There are a number of unnecessary steps being pulled in to this workflow.  For example, step `upi-conf-vsphere-ova` ensures that the required OVA for the installer associated with the CI job exists if it doesn't the OVA gets uploaded.  I don't see anything really problematic since the likelihood of the OVA not already existing is somewhat low.   It looks like steps could be omitted from the workflow:
- wicd-unit-vsphere-ovn-conf
- wicd-unit-vsphere-ovn-conf-hybrid-manifest-with-custom-vxlan-port
- wicd-unit-vsphere-upi-conf-vsphere-platform-none
- wicd-unit-vsphere-upi-conf-vsphere
- wicd-unit-vsphere-upi-conf-vsphere-ova

It might be a good idea to create a new chain which only pulls in the needed steps.

Reply:
I've cut down on the steps but would personally like to avoid creating a new chain, [see here for reasoning](https://github.com/openshift/release/pull/27251#discussion_r844355278)